docs/api/apiv3/endpoints/schemas.apib

```diff
@@ -49,10 +49,17 @@ Only one of the links (`allowedValues`, `allowedValuesSchemas`) will exist for a
 | regularExpression | The value of the property must match the given regular expression (if not null)    | String   | null    |
 | required          | If true this property is not nullable                                              | Boolean  | true    |
 | writable          | If false it is not allowed to **change** the properties value                      | Boolean  | true    |
+| location          | If present, contains a reference to the location of the property in the JSON       | String   |         |
 
 All of the above properties that do not have a default value *must* be present in the schema.
 For properties that have a default value, the client can assume the default value, if the property is missing.
 
+### Location property
+
+The location property gives a hint as to where to find the property if it is not contained in the set of local attributes.
+For example, for linked properties, the location attribute of the schema will contain `_links`. The property will then be found in the resource
+under the path `_links.propertyName`.
+
```

Comment:
I think it makes sense to explicitly explain the case where the property is to present which would mean that the property is to be placed within the top level properties or to let the property be always present with a `null` value in case of top level properties.